src/main/groovy/io/gatling/gradle/GatlingPluginExtension.groovy

```diff
@@ -12,6 +19,14 @@ class GatlingPluginExtension implements JvmConfigurable {
     private static final String API_TOKEN_ENV = "GATLING_ENTERPRISE_API_TOKEN"
     private static final String SIMULATION_ID_PROPERTY = "gatling.enterprise.simulationId"
     private static final String SIMULATION_ID_ENV = "GATLING_ENTERPRISE_SIMULATION_ID"
+    private static final String TEAM_ID_PROPERTY = "gatling.enterprise.teamId"
+    private static final String TEAM_ID_ENV = "GATLING_ENTERPRISE_TEAM_ID"
```

Comment:
**question:** should we really propose environment variable for all ? It was added only for api token for security purpose

Reply:
You're right, I'm keeping only the system props